### ISSUE TITLE
BitCode support

### ISSUE THREAD

**Issue by olarivain** [mixpanel/mixpanel-iphone#278]
Hey,

while most of the MP framework is open source, there is a part that links to a static .a (libMixpanelHelper.a I believe).
This library is built without bitcode support, which will make integration with watchOS 2 apps a significant headache: watchOS 2 makes bitcode mandatory, so at the moment, it's not possible to use mix panel in a watch app.
Through the magic of shared libraries, this can be transitively applied to the companion app.

In other words, a bitcode build of this library (or even better, the source code for it) is kind of required by september if mix panel customers want to ship watch apps. Is this something that's one the roadmap?

**Comment by kamoljan** [comment]
+1 
I am having the same issue for iOS 8.(3/4) devices on Xcode7.

**Comment by yuzaken** [comment]
+1 Same here

**Comment by samgreen** [comment]
This will be resolved with #288.

**Comment by samgreen** [comment]
This should be resolved now if you want to do a `pod install`.

**Comment by kamoljan** [comment]
@samgreen thank you. I will try to check it this weekend.

**Comment by kamoljan** [comment]
@samgreen `pod install` is still pointing to version `2.8.1`.

**Comment by samgreen** [comment]
@kamoljan Have you locked the version in your Podfile? I'm showing 2.8.2 when I run pod install in a new project.

**Comment by jaredmixpanel** [comment]
@kamoljan try `pod update` instead, this will ignore `Podfile.lock`. See here for further discussion: https://github.com/CocoaPods/CocoaPods/issues/760#issuecomment-46300500
